frontend/express/public/javascripts/pre-login.js

```diff
@@ -49,6 +49,47 @@ function encodeSomeHtml(html, options) {
     }
 }
 
+
+/**
+* Check password Validation.
+* @param {string} password - password string
+* @return {bool} result - return password is validated or not.
+* @memberof Pre Login
+* 
+/* exported passwordCheck */
+function passwordCheck(password) {
+    if (password.length < 8) {
+        return jQuery.i18n.prop("management-users.password.length", 8);
+    }
+    if (!/[A-Z]/.test(password)) {
+        return jQuery.i18n.map["management-users.password.has-char"];
+    }
+    if (!/\d/.test(password)) {
+        return jQuery.i18n.map["management-users.password.has-number"];
+    }
+    if (!/[^A-Za-z\d]/.test(password)) {
+        return jQuery.i18n.map["management-users.password.has-special"];
+    }
+    return true;
+}
+
+/**
+* Check form before submit.
+* @return {bool} result - return false to prevent submit action if check password failed.
+* @memberof Pre Login
+*
+/* exported validateForm */
+function validateForm() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['validateForm' is defined but never used.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=4805903)